code/game/machinery/vending/vendor_types/requisitions.dm

```diff
@@ -40,6 +40,7 @@
 		list("M56D Heavy Machine Gun", floor(scale * 2), /obj/item/storage/box/guncase/m56d, VENDOR_ITEM_REGULAR),
 		list("M2C Heavy Machine Gun", floor(scale * 2), /obj/item/storage/box/guncase/m2c, VENDOR_ITEM_REGULAR),
 		list("M240 Incinerator Unit", floor(scale * 2), /obj/item/storage/box/guncase/flamer, VENDOR_ITEM_REGULAR),
+		list("M707 Anti Material Rifle", floor(1), /obj/item/storage/box/guncase/vulture, VENDOR_ITEM_REGULAR),
```

Comment:
Why are you flooring 1

Reply:
because I don't know how to code and just removed the scaling :(

will attempt to fix

Reply:
fixed